### PR TITLE
LPS-165359

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-165359

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 1]
```diff
@@ -568,7 +568,17 @@ public ObjectField updateCustomObjectField(
...
+
+		if (StringUtil.equals(
+				newObjectField.getBusinessType(),
+				ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP)) {
+
+			newObjectField.setDBColumnName(name);
+		}
+		else {
+			newObjectField.setDBColumnName(name + StringPool.UNDERLINE);
+		}
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java:580]: Hi @p-albuquerque, could you explain a little more why we can't have the underline for relationship fields?

Don't we need to add an upgrade process to fix the existing columns?

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
@p-albuquerque As discussed via Slack, feel free to manually forward this PR to Brian.

Thanks!

**Comment by p-albuquerque** [comment]
resent here: https://github.com/brianchandotcom/liferay-portal/pull/124525

**Comment by p-albuquerque** [comment]
ci:close

**Pull request closed without merging**